payment/src/main/java/org/killbill/billing/payment/core/sm/payments/PaymentOperation.java

```diff
@@ -86,35 +82,16 @@ public OperationResult doOperationCallback() throws OperationException {
         } else {
             try {
                 return doSimpleOperationCallback();
-            } catch (final Exception e) {
-                // We need to unwrap OperationException (see doSimpleOperationCallback below)
-                throw unwrapExceptionFromDispatchedTask(e);
-            }
-        }
+	        } catch (final OperationException e) {
```

Comment:
![the-simpsons-episode-25 02-treehouse-of-horror-xxiv-promotional-photos-1_full](https://cloud.githubusercontent.com/assets/51940/15024500/fa2d898e-11e9-11e6-8b43-3bce5adcaab1.jpg)